src/js/components/stock-movement-wizard/outbound/PickPage.jsx

```diff
@@ -525,6 +535,60 @@ class PickPage extends Component {
       .catch(() => { this.props.hideSpinner(); });
   }
 
+  // Calculating which rows have quantity picked 0, and don't have subitems
+  getRowsWithEmptyPicks(pickPageItems) {
+    return pickPageItems.reduce((acc, item, index) => {
+      // When the quantity picked is equal to 0 we have to check its subitems,
+      // because the item can be edited to 0, not cleared.
+      if (!item.quantityPicked && !item.picklistItems.length) {
+        return [...acc, index + 1];
+      }
+
+      return acc;
+    }, []);
+  }
+
+  // Displaying an error message when an item with a quantity picked equal to 0 exists
+  showEmptyPicksErrorMessage(formValues) {
+    const emptyPicks = this.getRowsWithEmptyPicks(formValues.pickPageItems);
+    const emptyPickLinesNumber = emptyPicks.join(', ');
+    const alertMessage = this.props.translate(
+      'react.stockMovement.missingPickedLot.label',
+      `The picked lot is missing at rows: ${emptyPickLinesNumber}`,
+      {
+        rows: emptyPickLinesNumber,
+      },
+    );
+
+    this.setState({
+      alertMessage,
+      showAlert: emptyPicks.length,
+    });
```

Comment:
I assume this is currently done like this (using `showAlert` flag), but it feels weird to me that we are eventually assinging an `alertMessage` to the state and in the end we are not displaying it.
Imho from the "logical point of view" we should do:
```js
if (emptyPicks.length) {
  this.setState({
      alertMessage,
      showAlert: true,
    });
}
```
but since you are not the one introducing the `showAlert` flag, treat it just as whining